.travis.yml

```diff
@@ -4,11 +4,11 @@ services:
   - docker
 
 go:
-  - "1.13.x"
+  - "1.14.x"
 
 env:
   global:
-    - HELM_VERSION="v2.14.3"
+    - HELM_VERSION="v2.16.7"
```

Comment:
We currently test/package everything with 2.14.3. Is this change required? Should it be rolled out everywhere?

Reply:
Not required, I was updating all the package and updated that one as well. At this point we should be migrating everything to 3.x, but if 2.14 is the standard, I can set it back.

Reply:
Set this to v2.14.3, talked to MikeL about this issue. He will coordinate moving the Application Teams from 2.14 to 2.16 and long term to Helm 3.